src/gsi/mod_fv3_lola.f90

```diff
@@ -164,431 +168,641 @@ subroutine generate_anl_grid(nx,ny,grid_lon,grid_lont,grid_lat,grid_latt)
   real(r_kind) d(4),ds
   integer(i_kind) kk,k
 
+  integer(i_kind) name_len,nodeID,nodeComm,nodeRank,RanksPerNode,ierr,npes,inunit
+  character(len=72) :: input
+  character(len=5) :: np,nlatc,nlonc
```

Comment:
could more informative names be used for those character variables? like str_np, str_nlat ..?